app/src/app/list/list.component.sass

```diff
@@ -1,3 +1,7 @@
-.poll-list
+@import '../../sizes'
+
+h3
   margin-top: 3em
```

Comment:
The space between NavtreeComponent and the title (for instance on the home page) is now smaller than the space between mat-tab-group and the title. It should not be the case. Moreover, it would be better to use ex instead of em.